libs/authorizenet/src/actions/authorizenet.actions.ts

```diff
@@ -1,11 +1,12 @@
 import { Action } from '@ngrx/store';
 
 import { DaffAuthorizeNetTokenRequest } from '../models/request/authorize-net-token-request';
+import { DaffCartAddress } from '@daffodil/cart';
 
 export enum DaffAuthorizeNetActionTypes {
-  GenerateTokenAction = '[Daff-Authorize-Net] Generate Token',
-  GenerateTokenSuccessAction = '[Daff-Authorize-Net] Generate Token Success',
-	GenerateTokenFailureAction = '[Daff-Authorize-Net] Generate Token Failure',
+  UpdatePaymentAction = '[Daff-Authorize-Net] Generate Token',
+  UpdatePaymentSuccessAction = '[Daff-Authorize-Net] Generate Token Success',
+	UpdatePaymentFailureAction = '[Daff-Authorize-Net] Generate Token Failure',
```

Comment:
I think you should change the string values to be more descriptive as well.

Reply:
nah

Reply:
haha what?